sdk/formrecognizer/Azure.AI.FormRecognizer/src/AccountProperties.cs

```diff
@@ -16,11 +18,19 @@ internal AccountProperties(ModelsSummary_internal summary)
             LastUpdatedOn = summary.LastUpdatedDateTime;
         }
 
-        /// <summary> Current count of trained custom models. </summary>
+        /// <summary>
+        /// The current count of trained custom models.
+        /// </summary>
         public int CustomModelCount { get; internal set; }
-        /// <summary> Max number of models that can be trained for this subscription. </summary>
+
+        /// <summary>
+        /// The maximum number of models that can be trained for this subscription.
+        /// </summary>
         public int CustomModelLimit { get; internal set; }
-        /// <summary> Date and time (UTC) when the summary was last updated. </summary>
+
+        /// <summary>
+        /// The date and time (UTC) when the summary of all trained custom models was last updated.
+        /// </summary>
         internal DateTimeOffset LastUpdatedOn { get; set; }
```

Comment:
I thought we agreed to remove this property for preview 1? @annelo-msft

Reply:
That is correct.  We'll wait for a customer ask to add this back.